README.md

```diff
@@ -187,15 +187,14 @@ refund = Justifi::Refund.get(refund_id: 're_xyz')
 
 _Note: the term seller account has been deprecated and will be removed in future versions. Please use sub account instead_
 
-You can make requests using the `Seller-Account` header in order to process resources as a seller-account.
+You can make requests using the `Seller-Account` header (deprecated, use `Sub-Account` header) in order to process resources as a seller-account.
```

Comment:
WDYT?
```
You can make requests using the `Sub-Account` header (`Seller-Account` is deprecated) in order to process resources as a sub-account.
```

Reply:
sure

Reply:
Actually, a little below in the readme there is this statement: https://github.com/justifi-tech/justifi-ruby/blob/9f5ccaa0f3e59c6990ac466a45fe61fa1bdfe662/README.md#sub-account

So I added this just to note the header is deprecated in case someone didn't realize that